src/Stripe.net/Entities/StripeBalance.cs

```diff
@@ -5,6 +5,9 @@
 
     public class StripeBalance : StripeEntity
     {
+        [JsonProperty("object")]
+        public string Object { get; set; }
+
```

Comment:
Discovered that this class did not have `Object` so added it. Let me know if it should be a separate commit.

Reply:
👍 That's fine, thanks for the heads up!